.editorconfig

```diff
@@ -0,0 +1,12 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
```
end_of_line = lf
```

?

Reply:
no reason to standardize the endings locally as long as git checks them back in as LF

just annoying for windows users ![](https://up.clayton.cc/12ff3e.png)

Reply:
I was looking more towards people who upload files via GitHub. As it sounded like GitHub implemented EditorConfig natively but I couldn't find any documentation about this.